From 9d8d35534323fc2f3b253b422ef3ba54cf624f26 Mon Sep 17 00:00:00 2001
From: "Wang, ArvinX" <arvinx.wang@intel.com>
Date: Tue, 4 Sep 2018 14:03:14 +0800
Subject: [PATCH] Fix the time zone list is incomplete

The time zone list only display 20 items due to the PagedListView
limits the page number of list, the default number of pages is 5,
only 4 items can be displayed per page.

Set the default number of pages to unlimited pages.

Tracked-On: OAM-68129

Change-Id: Iae4979cf904695f358041ff46f56cfff7f8e17f5
Signed-off-by: Wang, ArvinX <arvinx.wang@intel.com>
---
 src/com/android/car/settings/common/ListItemSettingsFragment.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/com/android/car/settings/common/ListItemSettingsFragment.java b/src/com/android/car/settings/common/ListItemSettingsFragment.java
index 829b1c1..d200805 100644
--- a/src/com/android/car/settings/common/ListItemSettingsFragment.java
+++ b/src/com/android/car/settings/common/ListItemSettingsFragment.java
@@ -51,6 +51,7 @@ public abstract class ListItemSettingsFragment extends BaseFragment implements L
         PagedListView listView = getView().findViewById(R.id.list);
         listView.setAdapter(mListAdapter);
         listView.setDividerVisibilityManager(mListAdapter);
+        listView.setMaxPages(PagedListView.UNLIMITED_PAGES);
     }
 
     @Override
-- 
1.9.1

